.github/workflows/release.yml

```diff
@@ -119,6 +119,7 @@ jobs:
         run: make unit-test integration-test sqlness-test
 
       - name: Run cargo build
+        # TODO(discord9): remove `-F pyo3_backend` once make it default
```

Comment:
suggestion: you can use the following commands to specify the `-F pyo3_backend` (add the new step):

```
- name: Run cargo build with pyo3-backend
  if: contains(matrix.arch, 'linux') && endsWith(matrix.arch, '-gnu')
  run: cargo build ${{ matrix.opts }} --profile ${{ env.CARGO_PROFILE }} --locked --target ${{ matrix.arch }} -F pyo3_backend
```